### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by vkosuri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +32,11 @@ For examples, see the [examples](./examples) directory in this project's git rep
...
 nosetests
 ```
+
+## Motivation
+
+This repo is to address https://github.com/gunthercox/ChatterBot/issues/528
+
+## License
+
+ChatterBo IO Adapters t is licensed under the [BSD 3-clause license](./LICENSE).
```
Comment (gunthercox) [inline README.md]: It looks like there is a stray "t" after the word "Adapters".

**Review by gunthercox — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vkosuri) [reply README.md]: @gunthercox Thanks Master, i'll fix that

**Comment by vkosuri** [comment]
@gunthercox Master, Any suggestions/Comments?

**Comment by gunthercox** [comment]
Nope

**Pull request merged**